app/config.py

```diff
@@ -408,6 +408,28 @@ class Config(object):
             "schedule": crontab(hour=4, minute=0),
             "options": {"queue": QueueNames.PERIODIC},
         },
+        # app/celery/monthly_tasks.py
+        "resign-service-callbacks": {
+            "task": "resign-service-callbacks",
+            "schedule": crontab(minute=0)
+            if os.getenv("NOTIFY_ENVIRONMENT", "development") == "staging"
+            else crontab(day_of_month=3, hour=5, minute=0),
+            "options": {"queue": QueueNames.PERIODIC},
+        },
+        "resign-api-keys": {
+            "task": "resign-api-keys",
+            "schedule": crontab(minute=5)
+            if os.getenv("NOTIFY_ENVIRONMENT", "development") == "staging"
+            else crontab(day_of_month=3, hour=5, minute=15),
+            "options": {"queue": QueueNames.PERIODIC},
+        },
+        "resign-inbound-sms": {
+            "task": "resign-inbound-sms",
+            "schedule": crontab(minute=10)
+            if os.getenv("NOTIFY_ENVIRONMENT", "development") == "staging"
+            else crontab(day_of_month=3, hour=5, minute=30),
+            "options": {"queue": QueueNames.PERIODIC},
+        },
```

Comment:
Following Ben's recommendation: we could avoid configuration based on environment name and rather use feature flags. In this case though, we could have this tied to a configuration value (via an abstract method existing in children Config classes for example) that store either the cron value or an enum such as hourly or montly (to restrict values and mistakes).